src/components/queryBuilder/LimitEditor.tsx

```diff
@@ -1,14 +1,16 @@
 import React, { useState } from 'react';
 import { InlineFormLabel, Input } from '@grafana/ui';
-import { selectors } from './../../selectors';
+import labels from 'labels';
 
 interface LimitEditorProps {
   limit: number;
   onLimitChange: (limit: number) => void;
 }
+
 export const LimitEditor = (props: LimitEditorProps) => {
-  const [limit, setLimit] = useState(props.limit || 10);
-  const { label, tooltip } = selectors.components.QueryEditor.QueryBuilder.LIMIT;
+  const [limit, setLimit] = useState<number>(props.limit || 100);
```

Comment:
100 is too low. let's set at least 1000 as per https://docs.google.com/document/d/11IrTGUM62vuycbKbe6Et1TyKT-b6bCBhC2eJ2IthV8M/edit#heading=h.5inzegf5dtz5